File: Java/Introduction/src/Triangle.java
/*Write a program that computes the area of a right-angled triangle.*/
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Triangle {
    public static void main(String[] args) throws IOException {
        int length, width, height, area, perimeter;

        InputStreamReader inputStreamReader=new InputStreamReader(System.in);
        BufferedReader bufferedReader=new BufferedReader(inputStreamReader);

        System.out.println("Enter the length of the triangle");
        length=Integer.parseInt(bufferedReader.readLine());
        System.out.println("Enter the width of the triangle");
        width=Integer.parseInt(bufferedReader.readLine());
        System.out.println("Enter the height of the triangle");
        height=Integer.parseInt(bufferedReader.readLine());

        area=length*width*height;
        perimeter=length+width+height;

        System.out.println("The area of the triangle = "+area+" and the perimeter = "+perimeter);

    }
}

File: C#/Class exercises/Inheritance/Inheritance/triangle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Inheritance
{
    class triangle:shape
    {
        private int a, b, c;

        public void getdimensions()
        {
            Console.WriteLine("Enter the length");
            a= int.Parse(Console.ReadLine());
            Console.WriteLine("Enter the width");
            b = int.Parse(Console.ReadLine());
            Console.WriteLine("Enter the height");
            c = int.Parse(Console.ReadLine());

        }
        public double computeperimeter()
        {
            perimeter = a + b + c;
            return perimeter;

        }
        public double computearea()
        {
            area = a * b * c;
            return area;
        }
    }
}
